ch_pipeline/hfb/containers.py

```diff
@@ -338,3 +338,72 @@ class HFBHighResSpectrum(HFBHighResContainer):
             "distributed": False,
         },
     }
+
+
+class HFBRingMapBase(SiderealContainer, HFBContainer):
+    """Base class for HFB ringmaps."""
+
+    _axes = ("beam", "el")
+
+    @property
+    def el(self):
+        """The elevation in degrees associated with each sample of the elevation axis."""
+        return self.index_map["el"]
+
+
+class HFBRingMap(FreqContainer, HFBRingMapBase):
+    """Container for holding HFB ringmap data."""
+
+    _axes = ("subfreq",)
+
+    _dataset_spec = {
+        "hfb": {
+            "axes": ["freq", "subfreq", "beam", "el", "ra"],
+            "dtype": np.float32,
+            "initialise": True,
+            "distributed": True,
+            "distributed_axis": "freq",
+        },
+        "weight": {
+            "axes": ["freq", "subfreq", "beam", "el", "ra"],
+            "dtype": np.float32,
+            "initialise": True,
+            "distributed": True,
+            "distributed_axis": "freq",
+        },
+        "nsample": {
+            "axes": ["freq", "subfreq", "beam", "el", "ra"],
+            "dtype": np.uint16,
+            "initialise": False,
+            "distributed": True,
+            "distributed_axis": "freq",
+        },
+    }
+
+
+class HFBHighResRingMap(FreqContainer, HFBRingMapBase, HFBHighResContainer):
+    """Container for holding high-resolution frequency ringmap data."""
```

Comment:
I would maybe add a note in the documentation to emphasise that freq has moved to the back, and give some information as to why that's not insane.